src/modules/flight_mode_manager/tasks/Orbit/FlightTaskOrbit.hpp

```diff
@@ -44,6 +44,7 @@
 #include "FlightTaskManualAltitudeSmoothVel.hpp"
 #include <uORB/PublicationMulti.hpp>
 #include <uORB/topics/orbit_status.h>
+#include <uORB/topics/vehicle_command.h>
```

Comment:
`applyCommandParameters(const vehicle_command_s &command, bool &success)` also required `vehicle_command.h`, do you really need to re-include it here? How did it work before?